templates/nginx.conf

```diff
@@ -4,14 +4,32 @@ upstream pretalx_{{ pretalx_instance_identifier }}_server {
 
 proxy_cache_path /tmp/nginx-pretalx-{{ pretalx_instance_identifier }} levels=1:2 keys_zone=pretalx_static_{{ pretalx_instance_identifier }}:10m inactive=60m max_size=250m;
 
+{% if pretalx_tls %}
 server {
+    listen 80;
+    listen [::]:80;
+
+    server_name {{ pretalx_domain }}{% if pretalx_alternate_domains %} {{ pretalx_alternate_domains }}{% endif %};
+    return 301 https://{{ pretalx_domain }}$request_uri;
+}
+{% endif %}
+
+server {
+    {% if pretalx_tls %}
     listen 443 ssl;
     listen [::]:443 ssl;
+    {% else %}
+    listen 80;
+    listen [::]:80;
+    {% endif %}
+
     server_name {{ pretalx_domain }}{% if pretalx_alternate_domains %} {{ pretalx_alternate_domains }}{% endif %};
 
+    {% if pretalx_tls %}
     ssl on;
-    ssl_certificate /etc/ssl/letsencrypt/certs/{{ pretalx_domain }}/fullchain.pem;
-    ssl_certificate_key /etc/ssl/letsencrypt/certs/{{ pretalx_domain }}/privkey.pem;
+    ssl_certificate /etc/letsencrypt/archive/{{ pretalx_domain }}/fullchain1.pem;
```

Comment:
Please revert this change, as it will break the behaviour of this role for existing users.